src/sentry/integrations/slack/message_builder/disconnected.py

```diff
@@ -1,17 +1,24 @@
-from sentry.integrations.slack.message_builder import SlackBody
-from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
+from sentry.integrations.slack.message_builder import SlackBlock
+from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
 
 DISCONNECTED_MESSAGE = (
     "Slack has been uninstalled from your Sentry organization, re-install it to continue."
 )
 
 
-class SlackDisconnectedMessageBuilder(SlackHelpMessageBuilder):
-    def __init__(self) -> None:
-        """Override parent constructor."""
-        super().__init__()
+class SlackDisconnectedMessageBuilder(BlockSlackMessageBuilder):
+    def get_docs_block(self) -> SlackBlock:
```

Comment:
this is only used here, so moved from the old parent class. we also don't need to subclass SlackHelpMessageBuilder anymore